airbyte-integrations/connectors/source-goldcast-api/acceptance-test-config.yml

```
@@ -0,0 +1,37 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-goldcast-api:dev
+acceptance_tests:
+  spec:
+    tests:
+      - spec_path: "source_goldcast_api/spec.yaml"
```

```suggestion
      - spec_path: "source_goldcast/spec.yaml"
```

@FVidalCarneiro If you do end up using the `goldcast` for the name, make sure to rename `goldcast_api` and `goldcast-api` everywhere, including the folder name